ExtIO_sddc/RadioHandler.cpp

```diff
@@ -453,3 +467,18 @@ void RadioHandlerClass::UpdBiasT_VHF(bool flag)
 		hardware->FX3UnsetGPIO(BIAS_VHF);
 }
 
+void RadioHandlerClass::FineTuneLO(complexf* input, int nsample, int rd)
+{
+	if (GetFine_LO() && (glLOcorr != 0.0F)) // if Fine Local Oscillator Tuning allowed and required
```

Comment:
why not preventing set glOcrr to non-zero when FineLocalOSC is disabled? This will reduce the checks in couple of places.

Reply:
You are right as usual :-)